waspc/data/Generator/templates/server/src/auth/providers/email/signup.ts

```diff
@@ -1,150 +1,167 @@
-import { Request, Response } from 'express';
```

Comment:
This file was auto formatted, but the only changes are calls to the `onBeforeSignup`
and `onAfterSignup` hooks.